.mergify.yml

```diff
@@ -1,6 +1,13 @@
 # See https://doc.mergify.io
 
 pull_request_rules:
+  - name: label core
```

Comment:
Can we reuse this label in discard stale reviews below, so we don't have to maintain this twice?

Reply:
Nice idea.